docs/user-guide/zowe-secure-setup.md

```diff
@@ -0,0 +1,225 @@
+- [Multi-factor authentication (MFA)](#multi-factor-authentication-mfa)
+- [Authorization](#authorization)
+- High Availability
+  - [Sysplex architecture and configuration](#sysplex-architecture-and-configuration)
+  - [Caching service setup and configuration](#caching-service-setup-and-configuration)
+- Observation
+  - Audit Log 
+    - Integrate data in your observability solution
+- Distributed Identity Federation
+  - Okta
+  - KeyCloak
+
+## Transport Layer Security(TLS)
+
+The TLS protocol should be used to ensure secure data-transport for all connections to API Mediation Layer services. 
+
+### TLS requirements
+
+* Java in version at least 8 sr6 fp25 is installed on the system.
+* The following list shows the cipher suites that API ML services use.
+
+```
+TLS_AES_128_GCM_SHA256, TLS_AES_256_GCM_SHA384, TLS_CHACHA_POLY1305_SHA256
+```
+
+For more information, see the [TLS requirements in Zowe API ML requirements](../extend/extend-apiml/zowe-api-mediation-layer-security-overview#zowe-api-ml-tls-requirements).
+
+### SAF keyring
+
+You can choose to use a SAF keyring instead of keystore and truststore for storing certificates. It is more secure than PKCS12 files. And SAF keyring allows you to import existing or generate new certificates with Top Secret, ACF2, and RACF.
+
+For details about SAF Keyring, see the documentation [here](../extend/extend-apiml/certificate-management-in-zowe-apiml.md).
+
+## Authentication methods
+
+The API Mediation Layer provides multiple methods which clients can use to authenticate.
+
+### Authentication with JSON Web Tokens(JWT)
+
+When the client authenticates with the API ML, the client receives the JWT token in exchange. This token can be used for further authentication. The JWT secret that signs the JWT Token is an asymmetric private key that is generated during Zowe keystore configuration.
+
+To utilize [Single-Sign-On (SSO)](../user-guide/systemrequirements-zos/#single-sign-on-sso), the Zowe API ML client needs to provide the access token to API services in the forma of the cookie `apimlAuthenticationToken`, or in the `Authorization: Bearer` HTTP header as described [here](https://github.com/zowe/sample-spring-boot-api-service/blob/master/zowe-rest-api-sample-spring/docs/api-client-authentication.md#authenticated-request).
+
+- Validation:
+  - Use Gateway query endpoint to validate the token and retrieves the information associated with the token.
+  - Use JWKS to validate.
+
+- JWT token issuers
+  - API ML - SAF, zOSMF LTPA (Lightweight Third-Party Authentication)
+  - zOSMF - zOSMF JWT
+
+Encoded JWT:
+```
+eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+```
+
+**Note** The endpoint is disabled by default. For more information, see [Enable JWT token endpoint](../user-guide/api-mediation/api-gateway-configuration#enable-jwt-token-refresh-endpoint).
```

Comment:
> Looks like this note is not in the correct place, as the document didn't mention the JWT refresh end-point till now. Or maybe the token refresh functionality needs to be described just above the note.
Will remove this note here in case of confusing users.